js/src/annotations/osd-region-draw-tool.js

```diff
@@ -184,16 +201,6 @@
         _this.destroy();
       }));
 
-      this.eventsSubscriptions.push(_this.eventEmitter.subscribe('refreshOverlay.' + _this.windowId, function(event) {
```

Comment:
Yes, but what happens inside of this will likely be quite different, so it's better to re-code it when it's added rather than be confused by outdated code.